### ISSUE TITLE
Some help with your notes :)

### ISSUE THREAD

**Issue by usersaurus** [emmeryn/aero15-linux#1]
Ey! I'm a 15X owner too and I wanna share with u some adquired knowledge about all this time.
- Ubuntu 19.04 is working fine until now with `nouveau.modeset=0` in grub.
- Nvidia Prime working flawlessly out of the box. My previous experience was horrible with this, so I'm pretty surprised how fine is working now. I'm having a pretty good battery life.
![image](https://user-images.githubusercontent.com/14541427/56853672-56a9f580-692b-11e9-9c92-323189074c32.png)
- Display port, wifi, sound and aceleration working without issues.
- No way to make brightness shortcuts to work. xbacklight is not working too :(

**Comment by leoagomes** [comment]
While trying to fix it, I found out that the backlight keyboard shortcuts don't actually behave as keyboard keys. I guess it works on windows because of gigabyte's drivers. I also really wanted to be able to control fan speed on linux. :(

**Comment by usersaurus** [comment]
Yep. These kind of problems made me come back to Windows 10, and since the wls 2 release is coming soon I already don't see the need of keep fighting against the big amount of conflicts this laptop has with linux distros :(

**Comment by emmeryn** [comment]
whoops, I didn't see this! how good is your battery life? I haven't fiddled much with my Linux Mint install since my last update (I'm using it for work haha). I have a feeling that regardless what I do, the nVidia GPU is always on and that's why my battery life is so bad.

for the keyboard shortcuts, yeah, I have no idea too :(

**Comment by usersaurus** [comment]
Back to Ubuntu again X) 

This time all was incredibly smooth. Live boot with nomodeset, install with third party dependencies, remove usb, restart and everything was working out of the box (except the brightness shortcut, of course). I'm not having any problem at the moment and the battery life is quite good (between 6 and 8 hours).

**Comment by usersaurus** [comment]
Update

After 2 days working non-stop with it, I can confirm Ubuntu is working better than never before.

What's not working
- Brightness shortcut (but it can be modified with the top-right ubuntu slider)

What's working
- Everything else

Pretty glad at this moment :dancer:

**Comment by Zlisu** [comment]
> Update
> 
> After 2 days working non-stop with it, I can confirm Ubuntu is working better than never before.
> 
> What's not working
> 
> * Brightness shortcut (but it can be modified with the top-right ubuntu slider)
> 
> What's working
> 
> * Everything else
> 
> Pretty glad at this moment

Hey bro, are you able to make the fingerprint reader work?

**Comment by usersaurus** [comment]
> Hey bro, are you able to make the fingerprint reader work?

Aero 15X has not fingerprint reader. Anyway, I think there's no way to make fingerprint readers work in Ubuntu.

**Comment by leoagomes** [comment]
> Update
> 
> After 2 days working non-stop with it, I can confirm Ubuntu is working better than never before.
> 
> What's not working
> 
> * Brightness shortcut (but it can be modified with the top-right ubuntu slider)
> 
> What's working
> 
> * Everything else
> 
> Pretty glad at this moment 💃

Nice. I'll be reinstalling Ubuntu on my Aero 15 soon. Thermals lately have been awful (it gets really hot for no apparent reason). Has it ever been a problem/is it still one in your case?

**Comment by usersaurus** [comment]
My thermals are pretty ok afaik
![Screenshot from 2019-08-10 00-39-13](https://user-images.githubusercontent.com/14541427/62812740-6549a080-bb07-11e9-9b02-4f612b78d0a9.png)

**Comment by Tato14** [comment]
Hi! New Aero here. No problems with the set up except the brightness issue which I cannot control either with Fn keys (already known) nor the scroll in the top-right menu. Any hints on why is this happening?

**Comment by Zlisu** [comment]
> My thermals are pretty ok afaik
>

The Package id 0 and temp 1 get pretty high sometimes on my Aero 15 OLED. Wonder what temperatures these indicate.
![temperature](https://user-images.githubusercontent.com/17038536/65822993-df4ef980-e21b-11e9-8318-d732874d1822.png)

**Comment by ghost** [comment]
I have the XB 2020 running Ubuntu 20.10. I want to control the fan speeds but I'm not sure how since the ControlCenter application is only available for Windows. How can I do this? I know it's done through the embedded controller but there's no documentation whatsoever on it...

**Comment by ghost** [comment]
Well, I've thought a little bit and perhaps if I were to someone stick a windows installation onto a flash drive I could then make changes and have something else monitoring it. But how do I make a bootable Windows USB drive, and how do I make sure nothing changes between booting back into linux, or is there a way to monitor it on Windows?

Also, I just installed Pop! OS (https://pop.system76.com/) and it is a lot more friendly with my device (including brightness control without scripts or anything, @Tato14 !!). Working 'Fn+' keys for me are F1 (suspend), F7-F9 (volume) and F11 (airplane mode).
